pty/pty.go

```diff
@@ -47,15 +47,15 @@ type PTY interface {
 	// uses the output stream for writing.
 	//
 	// The same stream could be read to validate output.
-	Output() ReadWriter
+	Output() io.ReadWriter
```

Comment:
It's also preferable to return concrete types, accept interfaces rather than returning interfaces (unless completely required). See the [`ireturn`](https://github.com/butuzov/ireturn) linter and [why](https://github.com/butuzov/ireturn#reading-list) 😉